docs/admin/release_notes/version_2.5.md

```diff
@@ -0,0 +1,29 @@
+
+# v2.5 Release Notes
+
+This document describes all new features and changes in the release. The format is based on [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic
+Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Release Overview
+
+- Major features or milestones
+- Changes to compatibility with Nautobot and/or other apps, libraries etc.
```

Comment:
Let's update this to include an overview of the new features going into this version.